modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -430,6 +430,13 @@ R & t \\
     \f[u = f_x (x' + \alpha y') + c_x \\
     v = f_y y' + c_y\f]
 
+    Summary:
+    Generic camera model with perspective projection and without distortion correction
+
+    Kannala, Juho & Brandt, Sami. (2006). A Generic Camera Model and Calibration Method for
```

Comment:
Please add it latex bib file and add reference here. See https://github.com/opencv/opencv/blob/4.x/doc/opencv.bib

Reply:
Fixed